API/bin/Script.pgsql

```diff
@@ -63,8 +63,11 @@ INSERT INTO Games (title, category, rating, min_players, max_players, min_playti
 
 CREATE TABLE App_Admins(
 	id SERIAL PRIMARY KEY,
-	username VARCHAR(20) UNIQUE NOT NULL,
-	password VARCHAR(100) NOT NULL
+	email VARCHAR(20) UNIQUE NOT NULL,
+	hash VARCHAR(100) NOT NULL
 );
 
-INSERT INTO App_Admins (username, password) VALUES ('dev', 'admin'), ('wargamers', 'wARGamERs2018?');
+INSERT INTO App_Admins (email, hash) VALUES ('dev', 'admin'), ('wargamers', 'wARGamERs2018?'), ('test2', '$2b$10$77gEDFSVoXWz.7/IA6BT9.HUD2rWsUBkhuh0w6siKReqJalC9.71y')
```

Comment:
maybe you should remove the inserting of dev-admin and wargamers-wARGamERs2018? since they aren't hashes?